reactioncommerce/reaction
# feat: add read-only version of orders 2.0 to break up PR's

Part of #5157   
Impact: **major**  
Type: **feature**

## Issue
The PR #5157, which refactors the Catalyst Orders panel to use all GraphQL, as well as a new design, was getting very big. This PR breaks off a read-only version of the UI, which will help break the review into smaller pieces, as well as meet the need for a read-only version of this to be available for some end to end testing we are trying to do, without waiting on the full functionality, which isn't needed for this testing.

## Breaking changes
None. The old Orders UI is still the primary UI, and you need a special URL to access the new Panel.

## Testing
1. Create an order, with multiple items and multiple payments, via the storefront
1. Break the Items into multiple fulfillment groups using the addOrderFulfillmentGroup mutation ( #5027 ) 
1. See that you can see the order in the "Classis" Orders UI by going to the orders page, clicking on the order from the Table.
1. Change the url from `/orders/`, to `/orders-2.0/`, to see the order in the new UI
1. See that all the data is correct, that the fulfillment groups are broken up

** please note translations for order statuses will not be added by default to a new shop. you can follow instructions here on how to add them, if you wish: https://github.com/reactioncommerce/reaction/pull/4981
<img width="342" alt="Order_Details_for_order_reference__BpkeapiCwDpKYmJL5" src="https://user-images.githubusercontent.com/4482263/59006889-08fd9400-87d9-11e9-8cf4-73c034e02489.png">
